src/modules/delete.js

```diff
@@ -5,12 +5,16 @@ export let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
 export default function deleteTask() {
   const deleteBtn = document.querySelectorAll('.option');
   deleteBtn.forEach((btn) => {
-    const container = document.querySelector('.container');
     btn.addEventListener('click', (e) => {
+      const deletedTaskIndex = e.target.dataset.index;
       const item = e.target.parentNode.parentNode;
-      container.removeChild(item);
-      const deletedTask = e.target.previousElementSibling.textContent;
-      tasks = tasks.filter((task) => task.description !== deletedTask);
+      item.remove();
+      tasks.splice(deletedTaskIndex, 1);
+
```

Comment:
- Just like the previous reviewer suggested, your app doesn't work as expected, first according to the project requirements, when a user clicks on the checkbox it should be updated in the local storage i.e `completed: true`.
<img width="960" alt="trjsrjsrt" src="https://user-images.githubusercontent.com/73607512/214547202-ab750a2f-c7fa-4e5a-b011-56221467899c.PNG">
- Also, when I click on the `clear completed` button, according to the project requirements, all completed tasks should be removed from the DOM, and all indexes should be updated. Please fix this 👍 👍 

**OPTIONAL** - I noticed that when I delete any tasks, asides from the last task to be added to the DOM, all tasks in the local storage get deleted. Please fix this 🙏 
<img width="960" alt="bfbsfsdfv" src="https://user-images.githubusercontent.com/73607512/214546998-b0f1fa43-8fb4-4075-b988-d34a48e806a6.PNG">